client/src/assets/common-styles/_common.sass

```diff
@@ -0,0 +1,36 @@
+@import 'helpers/frompxtorem'
+
+*
+    box-sizing: border-box
+    margin: 0
+    padding: 0
+.flex
+    display: flex
+    justify-content: center
+    align-items: center
+
+.flex-col
+    display: flex
+    flex-direction: column
+    justify-content: center
+    align-items: center
+
+.container
+    width: 80vw
+
+.grid
+    display: grid
+    justify-items: center
+    align-content: center
+.blue-button
```

Comment:
usually we use something like `primary`, `danger`, `warning` instead of concrete colors